### DISCUSSION TITLE
Auto import dividends

### DISCUSSION THREAD

**Discussion by zaromarco** [ghostfolio/ghostfolio#3418] [Ideas]
Hi , i'm using ghostfolio for a while , have added many activities and manually import the dividends after the date i bought stock bond and so on.
doing again the manual import i saw that ghostfolio knows the already imported dividends and automatically select the new one not already imported , so far great but it would be very usefull if ghostfolio autoimport dividends of the portfolio automatically after the activities were created or after the first manual dividends import is done. Now i need every day to manually redo the import

**Comment by swissbuechi** [comment]
Just make sure to allow this feature to be disabled.
I often need to track it manually, since I have the same holding spread across different accounts.
Also, the amount of dividends is rounded up and does not match the number reported on IBKR. So automatically is not an option for everyone ATM.
